KIPs/kip-17-non_fungible_token.md

```diff
@@ -0,0 +1,544 @@
+---
+kip: 17
+title: Non-fungible Token Standard
+author: Junghyun Colin Kim <colin.kim@groundx.xyz>
+discussions-to: https://github.com/klaytn/kips/issues/17
+status: Draft
+type: Standards Track
+category: Token
+created: 2020-03-04
+requires: 13
+---
+
+## Simple Summary
+A standard interface for non-fungible tokens (NFTs), also known as deeds.
+
+## Abstract
+The following standard allows for the implementation of a standard API for NFTs within smart contracts. This standard provides basic functionality to track and transfer NFTs.
+
+We considered use cases of NFTs being owned and transacted by individuals as well as consignment to third party brokers/wallets/auctioneers ("operators"). NFTs can represent ownership over digital or physical assets. We considered a diverse universe of assets, and we know you will dream up many more:
+
+- Physical property — houses, unique artwork
+- Virtual collectibles — unique pictures of kittens, collectible cards
+- "Negative value" assets — loans, burdens and other responsibilities
+
+In general, all houses are distinct and no two kittens are alike. NFTs are *distinguishable* and you must track the ownership of each one separately.
+
+## Motivation
+A standard interface allows wallet/broker/auction applications to work with any NFT on Klaytn. We provide simple KIP-17 smart contracts as well as contracts that track an *arbitrarily large* number of NFTs. Additional applications are discussed below.
+
+## Specification
+This document is heavily derived from [ERC-721](https://eips.ethereum.org/EIPS/eip-721) written by William Entriken, Dieter Shirley, Jacob Evans, and Nastassia Sachs.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+### Differences from ERC-721
+This section describes the differences between KIP-17 and ERC-721. 
+
+- Every token transfer/mint/burn MUST be tracked by event logs. This means that a Transfer event MUST be emitted for any action related to transfer/mint/burn.
+- KIP-17 also supports the wallet interface of ERC-721 (`ERC721TokenReceiver`) to be compliant with ERC-721.
+- More optional extensions are defined (minting extension, minting with URI extension, burning extension, and pausing extension).
+
+### KIP-13 Identifiers
+The below table shows KIP-13 identifiers for interfaces defined in this proposal.
+
+|Interface|KIP-13 Identifier|
+|---|---|
+|[KIP17](#kip17-interface)|0x80ac58cd|
+|[KIP17TokenReceiver](#wallet-interface-kip17tokenreceiver)|0x6745782b|
+|[ERC721TokenReceiver](#wallet-interface-kip17tokenreceiver)|0x150b7a02|
+|[KIP17Metadata](#metadata-extension)|0x5b5e139f|
+|[KIP17Enumerable](#enumeration-extension)|0x780e9d63|
+|[KIP17Mintable](#minting-extension)|0xeab83e20|
+|[KIP17MetadataMintable](#minting-with-uri-extension)|0x50bb4e7f|
```

Comment:
How about checking if this value is correct?

Reply:
Oh, it needs to be updated. Thanks for the comment.